### ISSUE TITLE
numpy.fromstring deprecation warning

### ISSUE THREAD

**Issue by sirspinach** [wookayin/tensorflow-plot#9]
I'm using tfplot to write a histogram figure to TensorBoard. If I import `tensorflow_probability` I get the following warning:

```
/usr/local/lib/python3.5/dist-packages/tfplot/figure.py:84: DeprecationWarning: The binary mode of fromstring is deprecated, as it behaves surprisingly on unicode inputs. Use frombuffer instead
  img = np.fromstring(fig.canvas.tostring_argb(), dtype=np.uint8, sep='')
```

Curiously I'm not even using any packages from `tensorflow_probability` right now. But as soon as I import this module, I start getting this warning.

**Comment by sirspinach** [comment]
Using frombuffer instead suppresses the warning:
```
img = np.frombuffer(fig.canvas.tostring_argb(), dtype=np.uint8)
```

Another fix would be to suppress the warning with 
```
with numpy.warnings.catch_warnings():
    numpy.warnings.filterwarnings('ignore', r'DeprecationWarning: The binary mode of fromstring is deprecated, as it behaves surprisingly on unicode inputs. Use frombuffer instead')
```

**Comment by wookayin** [comment]
Resolved in #10. Thanks a lot!
